Dockerfile

```diff
@@ -6,6 +6,9 @@ ENTRYPOINT ["/bin/git-contrib-graph"]
 RUN apk add --no-cache git
 
 COPY git-contrib-graph.go /app/
+COPY go.mod /app/
```

Comment:
what about `go.sum` ?

Reply:
Good call, while not strictly speaking necessary it's probably better to include it. Updated the PR.